### PR TITLE
Add SLO metrics

### PR DESCRIPTION
**Pull request by tonymeng**
Hello @dokipen, @matkam, 

Please review the following commits I made in branch 'tonymeng/add-timestamps':

- **picchu_git_create_latency** (0ebdb174f9edfc060da4f9155f3ff3de0c9e538c)

- **picchu_incarnation_oldest_age_seconds** (2fc5e5c19a677a2cfe30b3f0c9a744e00b46c9ca)

- **Setup timestamps** (fc5a0a545280f84cbbb557f190bc911ffc6d0d71)

R=@dokipen
R=@matkam

### CODE REVIEW

File: pkg/apis/picchu/v1alpha1/releasemanager_types.go [hunk 1]
```diff
@@ -52,7 +52,10 @@ type ReleaseManagerRevisionStatus struct {
 	LastUpdated          *metav1.Time                        `json:"lastUpdated"`
 	GitTimestamp         *metav1.Time                        `json:"gitTimestamp,omitempty"`
 	RevisionTimestamp    *metav1.Time                        `json:"revisionTimestamp,omitempty"`
+	IncarnationCreated   *metav1.Time                        `json:"incarnationCreationTimestamp,omitempty"`
 	CanaryStartTimestamp *metav1.Time                        `json:"canaryStartTimestamp,omitempty"`
+	IncarnationDeployed  *metav1.Time                        `json:"incarnationDeployedTimestamp,omitempty"`
+	IncarnationReleased  *metav1.Time                        `json:"incarnationReleasedTimestamp,omitempty"`
```
Comment (dokipen) [inline pkg/apis/picchu/v1alpha1/releasemanager_types.go]: Why do we care about incarnation times? Isn't this leaking the impl? If this is purely for debugging, it's fine, but I don't think it should be an SLI.

Comment (tonymeng) [reply pkg/apis/picchu/v1alpha1/releasemanager_types.go]: basically im trying to measure time to initial ramp from when picchu knows about a revision to the time customers are using that revision. I'll rename it `DeployedTimestamp` and `ReleasedTimestamp`

Comment (dokipen) [reply pkg/apis/picchu/v1alpha1/releasemanager_types.go]: I think revision's creationTimestamp will be more accurate

**Comment by tonymeng** [comment]
⚡️

**Pull request merged**